lib/transform.js

```diff
@@ -14,15 +14,9 @@ module.exports = (image,
     crop = false,
     cropMaxSize = 2000,
     gravity = sharp.gravity.center,
-    format = sharp.format.jpeg.id,
-    progressive = true,
-    quality = 80
```

Comment:
What are we losing here?

Reply:
a bug :) 

But in all seriousness, without this change whether you serve .webp or .png its always being formatted to jpeg. I also think that this package must have only the minimal amount of "hardcoded" config. quality or progressive should be set as an optional field in `makeUrl` just like height and width. I've removed both of them here because they weren't affecting the actual image it seemed.

Reply:
`progressive` affects the perceived performance psotively (vs `baseline` by default).

Reply:
That's right, therefore it should be included as an optional field inside makeUrl and be set progressive by default.